app/services/task-counter.js

```diff
@@ -0,0 +1,28 @@
+const getAllTasks = require('./data/get-all-tasks')
+const taskStatus = require('../constants/task-status')
+const markCurrentWorkloadAsComplete = require('./data/update-current-workload-report-with-status')
+const callWebRefreshEndpoint = require('./refresh-web-org-hiearchy')
+
+module.exports = function (currentTask) {
+  console.log(currentTask)
+  var taskList = getAllTasks()
+  var numPending = taskList.filter(statusFilter('P')).length
```

Comment:
This will iterate through the task list 3 times (once per filter). We could have one loop and increment the counters accordingly for performance reasons.